### DISCUSSION TITLE
"Private" or "hidden" database fields

### DISCUSSION THREAD

**Discussion by AnandChowdhary** [prisma/prisma#2127] [Q&A] [answered]
Is there an implementation of "hidden" or "private" columns in the database for sensitive information? 

For example, if I request `prisma.users.findOne({ where: { id: 1 } })`, the resulting user can have the field `password` automatically deleted from the object.

Perhaps we can have an option like the following to keep all "hidden" fields, which can be modeled using something like `@hidden`:

```ts
const user = prisma.users.findOne({ where: { id: 1 }, include: { hidden: true } })
```

**Comment by Sytten** [comment, marked as answer]
This is a feature request, its not implemented. Not sure you really want that at the database layer, its more of a nexus request really.

**Reply by pantharshit00** [reply]
I also agree. @AnandChowdhary Please open a new feature request via https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title= as we can't convert a discussion into an issue. After that, we can see if this is something viable or not.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
